SQL/0000-00-00-schema.sql

```diff
@@ -1798,21 +1798,23 @@ CREATE TABLE `final_radiological_review_history` (
 -- Table structure for table `gene`
 DROP TABLE IF EXISTS `gene`;
 CREATE TABLE `gene` (
-  `GeneID` bigint(20) NOT NULL AUTO_INCREMENT,
+  `GeneID` int(20) NOT NULL AUTO_INCREMENT,
```

Comment:
Since you are changing the column type, I would suggest to use 'unsigned integer' instead of int(x). integer type allow 4 bytes of information (4294967295 if unsigned) which is plenty. Also, the 'x' in int(x) is only to tell MySQL to display this number on 20 characters.